Python/Python.sublime-syntax

```diff
@@ -15,38 +16,205 @@ file_extensions:
+            - include: expressions
+        - include: expressions
+    - match: \b(assert)\b
+      scope: keyword.other.assert.python
+      push:
+        - meta_scope: meta.statement.assert.python
+        - match: $
+          pop: true
+        - include: expressions
+    - match: \b(del)\b
+      scope: keyword.other.del.python
+      push:
+        - meta_scope: meta.statement.del.python
+        - include: line_continuation_or_pop
+        - include: expressions
+    - match: \b(print)\b(?! *($|[,.()\]}]))
+      scope: keyword.other.print.python
+      push:
+        - meta_scope: meta.statement.print.python
+        - include: line_continuation_or_pop
+        - include: expressions
+    - match: \b(exec)\b(?! *($|[,.()\]}]))
+      scope: keyword.other.exec.python
+      push:
+        - meta_scope: meta.statement.exec.python
+        - include: line_continuation_or_pop
+        - include: expressions
+    - match: \b(return)\b
+      scope: keyword.control.flow.return.python
+    - match: \b(break|continue|pass)\b
+      scope: keyword.control.flow.python
+
+  imports:
+    - match: \b(import)\b
+      scope: keyword.control.import.python
+      push:
+        - meta_scope: meta.statement.import.python
+        - include: line_continuation_or_pop
+        - include: import_alias
+        - include: dotted_name
+        - match: ','
+          scope: punctuation.separator.import-list.python
+    - match: \b(from)\b
+      scope: keyword.control.import.from.python
+      push:
+        - meta_scope: meta.statement.import.python
+          meta_content_scope: meta.import-source.python
+        - include: line_continuation_or_pop
+        - match: \b(import)\b
+          scope: keyword.control.import.python
+          set:
+            - include: line_continuation_or_pop
+            - match: ' *(\()'
+              captures:
+                1: punctuation.definition.begin.import-list
+              set:
+                - meta_scope: meta.statement.import.python
+                - include: comments
+                - match: \)
+                  scope: punctuation.definition.end.import-list
+                  pop: true
+                - include: import_name_list
+            - match: ''
+              set:
+                - meta_scope: meta.statement.import.python
+                - include: line_continuation_or_pop
+                - include: import_name_list
+        - include: dotted_name
+
+  import_name_list:
+    - match: ','
+      scope: punctuation.separator.import-list.python
+    - include: import_alias
+    - include: name
+    - match: \*
+      scope: keyword.other.import-all.python
+    - match: \S+
+      scope: invalid.illegal.name.import.python
+
+  import_alias:
+    - match: \b(as)\b
+      scope: keyword.control.import.as.python
+
+  block_statements:
+    # async for ... in ...:
+    - match: \b(async +)?(for)\b
+      captures:
+        1: keyword.control.flow.async.python
```

Comment:
`async` should probably be `storage.modifier` here. Having some instances of `async` (before `def`) be one scope and others be a different scope would feel very off to me as a user. I think that `storage.modifier` is more correct than `keyword.control` since `async` is modifying how the `for` loop is being executed.